### DISCUSSION TITLE
mount: /mnt/xyz: unknown filesystem type 'juicefs'

### DISCUSSION THREAD

**Discussion by serhiicherepanov** [juicedata/juicefs#4374] [General]
I added the line to my fstab
```fstab
mysql://juicefs:juicefs@(10.110.0.1:3306)/mnt_xyz  /mnt/xyz  juicefs  _netdev  0 0
```
When I try to mount the folder by following command
```bash
mount /mnt/xyz
```
The error appears 
```bash
mount: /mnt/xyz: unknown filesystem type 'juicefs'.
       dmesg(1) may have more information after failed mount system call.
```

I tried both install methods, by a script and manually.

**Environment**:
- `juicefs version 1.1.1+2023-11-28.437f4e6`
- `Debian GNU/Linux 12 (bookworm)`
- `Linux pve-ge-01 6.5.11-4-pve #1 SMP PREEMPT_DYNAMIC PMX 6.5.11-4 (2023-11-20T10:19Z) x86_64 GNU/Linux`
- `ZFS`
- `mariadb 10.11.4`
- `network: linux bridge`

**Comment by solracsf** [comment]
A symlink needs to be created from `/sbin/mount.juicefs` to the JuiceFS executable, e.g.
`ln -s /usr/local/bin/juicefs /sbin/mount.juicefs`

**Reply by serhiicherepanov** [reply]
@solracsf 

Thanks, because I moved this setup from another server I forgot about first mount with the `--update-fstab`flag. When I did it the binary was symlinked properly.
